tools/wptrunner/wptrunner/testloader.py

```diff
@@ -562,6 +564,11 @@ def _load_tests(self):
                  "disabled":defaultdict(list)}
 
         for test_path, test_type, test in self.iter_tests():
+            if test.jsshell:
+                executor_class = self.executor_classes.get(test_type)
```

Comment:
It doesn't feel right to me that teastloader needs to know about executors. Can we avoid that?